client/dashboard/data/domain.ts

```diff
@@ -14,3 +15,9 @@ export function fetchDomain( domainName: string ): Promise< Domain > {
 		apiVersion: '1.2',
 	} );
 }
+
+export function disconnectDomain( domainName: string ): Promise< void > {
+	return wpcom.req.get( {
```

Comment:
This should be a post request (I checked the endpoint). I don't know why the existing feature is using a get 😅